api/v1beta1/oscmachine_validation.go

```diff
@@ -183,10 +183,12 @@ func ValidateVolumeType(volumeType string) (string, error) {
 	}
 }
 
+var isValidSubregion = regexp.MustCompile(`(cloudgouv-)?(eu|us|ap)-(north|east|south|west|northeast|northwest|southeast|southwest)-[1-2][a-c]`).MatchString
```

Comment:
Why not an array?

Here, it will validate `eu-west-1a` or `eu-west-2c`

cc @jfbus

Reply:
The goal was not to force users to upgrade whenever a new region/subregion is added. This is an interim fix with a better validation than before, aiming a low-risk inclusion in v0.4.0.

The right way would be to call the API to list all available subregions, but this would require a much larger refactoring.